src/quo2/components/community/banner/style.cljs

```diff
@@ -2,18 +2,21 @@
   (:require [quo2.foundations.colors :as colors]))
 
 (defn community-card
-  [radius]
-  {:shadow-offset    {:width  0
-                      :height 2}
-   :shadow-radius    radius
-   :shadow-opacity   1
-   :shadow-color     colors/shadow
-   :elevation        1
-   :border-radius    radius
-   :justify-content  :space-between
-   :background-color (colors/theme-colors
-                      colors/white
-                      colors/neutral-90)})
+  []
+  {:shadow-offset     {:width 0 :height 2}
+   :shadow-radius     16
+   :shadow-opacity    1
+   :shadow-color      colors/shadow
+   :elevation         1
+   :border-radius     16
+   :justify-content   :space-between
+   :background-color  (colors/theme-colors colors/white colors/neutral-90)
```

Comment:
Not exactly a concern for your PR, but since you're touching the banner component, you could make it themeable by using `with-theme` and passing the `theme` parameter to `colors/theme-colors`.

Reply:
Done :+1: